thirdparty/fluid/modules/gapi/include/opencv2/gapi/util/variant.hpp

```diff
@@ -509,6 +509,10 @@ namespace util
         return v.index() == util::variant<Types...>::template index_of<T>();
     }
 
+#ifdef __linux__
```

Comment:
ifdef __GNUC__

Reply:
Updated.
Changed macro __linux__ to __GNUC__.   
Judging macro **GNUC** don't work.